src/_cffi_src/utils.py

```diff
@@ -80,9 +82,18 @@ def build_ffi(module_name, cdef_source, verify_source, libraries=[],
 
 
 def extra_link_args(platform):
-    if platform != "win32":
+    if platform != "win32" or not _compiler_is_msvc():
```

Comment:
Is there a scenario where the compiler is msvc but the platform isn't win32? Seems kinda redundant.